src/components/TrackingDisabler.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from "react";
+
+import { EmbedTrackingProps } from "../types";
+
+interface Props extends EmbedTrackingProps {
+  disable?: boolean;
+}
+
+const TrackingDisabler = ({ id, disable = false }: Props) => {
+  const [scriptKey, setScriptKey] = useState(0);
+  useEffect(() => {
+    setScriptKey((num) => num + 1);
+  }, [disable, id]);
```

Comment:
This is needed so a fresh new script element is added (and executed) in the DOM whenever the `disable` option fires.